### PR TITLE
fix: media dimension in nft screen

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
https://showtime-rq88331.slack.com/archives/C02PXGK3V8D/p1663072878660519

### CODE REVIEW

File: packages/app/components/media/index.tsx [hunk 1]
```diff
@@ -80,8 +80,8 @@ function Media({
             style={sizeStyle}
             data-test-id={Platform.select({ web: "nft-card-media" })}
             blurhash={item?.blurhash}
-            width={size}
-            height={size}
+            width={sizeStyle?.width ?? size}
```
Comment (alantoa) [inline packages/app/components/media/index.tsx:83]: I have added this to 57 lines, is it necessary here? (maybe you missed it.) 😄
image

Comment (alantoa) [reply packages/app/components/media/index.tsx:83]: maybe my bad, it seems to miss height, but not sure it if is height cause of cropped media.  🤔

**Review by alantoa — APPROVED** [review]
LGTM ✨

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

File: packages/app/components/media/index.tsx (hunk 1 above)
Comment (intergalacticspacehighway) [reply packages/app/components/media/index.tsx:83]: yeah, height was causing it. I think on some NFTs it has different height and width!

**Pull request merged**